playbooks/robusta_playbooks/pod_enrichments.py

```diff
@@ -2,9 +2,11 @@
 
 from datetime import datetime
 
+from robusta.core.playbooks.prometheus_enrichment_utils import XAxisLine
```

Comment:
We don't want to assume it's the same code base as the runner.
If we need something from core, we need to add it to the robusta.api